src/styles/Signup.scss

```diff
@@ -0,0 +1,22 @@
+.sign-up-wrapper {
+  margin-top: 2rem;
+  text-align: center;
+
+  > * { display: block; }
+  > * + * { margin-top: 1rem; }
+}
+
+.sign-up-form {
+  max-width: 50rem;
+  width: 100%;
+
+  > * + * {
+    margin-top: 2.5rem;
```

Comment:
If you have only one property do it in one line, or not, but let them all be consistent across files.